ddtrace/contrib/pylons/compat.py

```diff
@@ -0,0 +1,5 @@
+try:
+    from pylons.templating import render_mako  # noqa
+    legacy_pylons = False
```

Comment:
required because we have a breaking change after 0.9.7

Reply:
Would be nice explicating more strictly how tight to a version it is (at least leaving a comment about Pylons' version)